verify_puzzle_stats.py

```
@@ -0,0 +1,42 @@
+stats.save()
+
+# Use Django test client to hit the endpoint
+client = Client()
+client.force_login(user)
+
+response = client.get(reverse('puzzle_stats'))
+
+print(f"Status Code: {response.status_code}")
+try:
+    print(f"Response JSON: {json.dumps(response.json(), indent=2)}")
+except Exception as e:
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)